### DISCUSSION TITLE
Cual es link para poder subir nuestra practica de la semana 1

### DISCUSSION THREAD

**Discussion by MiguelAngelAguilar312** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#132] [Q&A] [answered]
Buenos días compañeros saben cual es link para poder subir la información de la practica de esta semana.

**Comment by armap99** [comment]
Tengo entendido que lo van a subir el día de hoy así que solo nos queda esperar.

**Reply by MiguelAngelAguilar312** [reply]
Gracias

**Comment by armap99** [comment]
Ya esta disponible el formulario en teams en el apartado de la semana 1 por si gustas revisarlo compañero.

**Reply by dalexez** [reply]
Hola @armap99, una duda. En el post que hiso el MC de el link para subir las practicas alguien comento una fecha limite el 4 de marzo? cambio la fecha limite de entrega?

**Reply by armap99** [reply]
Es correcto por petición general se movió al 4 de marzo

**Reply by dalexez** [reply]
Ya lo vi jeje olvide dar click en "ver mas" gracias! 👍🏾

**Comment by romarpla** [comment, marked as answer]
[Link de publicación de Teams](https://teams.microsoft.com/l/message/19:8a61b3259c3147259ba0197e9b103432@thread.tacv2/1645814720628?tenantId=4ae54b05-b77e-4224-aef1-8661422e0816&groupId=d38ceaac-3599-4f30-94ba-740ccb530cb0&parentMessageId=1645814720628&teamName=Misi%C3%B3n%20Frontend%20-%20JS&channelName=Semana%201%20-%20Despegue%20%F0%9F%9A%80&createdTime=1645814720628)

Este es el link dentro del equipo de Teams para que puedan ver el link del Formulario
